src/components/internal/ContextualModal.tsx

```diff
@@ -12,7 +12,7 @@ export function ContextualModal(props: ContextualModalProps) {
   const { content, title } = props;
   const tid = useTestIds(props, "popup");
   return (
-    <FocusScope restoreFocus autoFocus>
+    <FocusScope restoreFocus>
```

Comment:
**thought:** My concern with removing the auto-focus behavior here is that for keyboard users they'd have to tab through the whole underlying screen before they can tab into this contextual modal now. Wondering if we can do this another way. I think the reason why we didn't run into this auto-focus issue on the Modal is because the first thing that gets focus is the close button.

Reply:
I'm going to create a storybook example in here and see how it behaves.